enterprise/users/src/test/java/io/crate/integrationtests/PrivilegesIntegrationTest.java

```diff
@@ -170,7 +170,41 @@ public void testQuerySysShardsReturnsOnlyRowsRegardingTablesUserHasAccessOn() th
     }
 
     @Test
-    public void testQuerySysAllocationsReturnsOnlyRowsRegardingTablesUserHasAccessOn() throws Exception {
+    public void testQuerySysJobsLogReturnsLogEntriesRelevantToUser() {
```

Comment:
I'm wondering if instead it would make sense to have a unit test for the concrete `StaticTableDefinition` ?

We've full integration tests already to make sure that the filtering works when everything is integrated.

Reply:
as discussed let's maybe keep one integration test for sys.jobs_log